source/index-management.rst

```diff
@@ -71,7 +71,18 @@ All drivers MUST offer at least one of the sections of operations, the Standard
 Operation Parameters
 --------------------
 
-All drivers MUST include the specified parameters in each operation, with the exception of the options parameter which is OPTIONAL. As of 3.4 (see https://jira.mongodb.org/browse/SERVER-769) the server validates options passed to the createIndexes command -- drivers should be aware when testing that passing arbitrary options when the driver does not validate them could fail on the server.
+All drivers MUST include the specified parameters in each operation. This does not preclude a driver from offering more. A driver SHOULD NOT require a user to specify the options parameter if they wish to use the server defaults.
+
+As of 3.4 (see https://jira.mongodb.org/browse/SERVER-769) the server validates options passed to the createIndexes command -- drivers should be aware when testing that passing arbitrary options when the driver does not validate them could fail on the server.
```

Comment:
nit: add backticks around `createIndexes` since its a constant (I have a mild preference for making these specifications deeply interlinked, or at least making things like this visually distinct)

Reply:
I couldn't both backtick and link due to how links are specified in rst, so I just backticked it. 

As an aside, would there be any support behind a proposal to allow new specifications to be written in markdown? It seems like the only place rst is common is in the python ecosystem and the mongodb driver specifications.

Reply:
:+1:

I think we'd need a DRIVERS ticket to discuss that. IIRC the preference for RST here was for the table of contents, which Markdown doesn't make for you automatically iirc.